src/panels/GettingStartedPanel/GettingStartedPanel.tsx

```diff
@@ -1,19 +1,28 @@
 import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
 import { Button, Group, Stack } from '@mantine/core';
 
 import { Layout } from '@/components/Layout/Layout';
 import { ArrowLeftIcon, ArrowRightIcon } from '@/icons/icons';
 import { useWindowLayoutProvider } from '@/windowmanagement/WindowLayout/hooks';
 
-import { ContentSteps } from './Steps/ContentSteps';
-import { IntroductionSteps } from './Steps/IntroductionSteps';
-import { NavigationSteps } from './Steps/NavigationSteps';
-import { TimeSteps } from './Steps/TimeSteps';
+import { useContentSteps } from './Steps/ContentSteps';
+import { useIntroductionSteps } from './Steps/IntroductionSteps';
+import { useNavigationSteps } from './Steps/NavigationSteps';
+import { useTimeSteps } from './Steps/TimeSteps';
 import { Chapters } from './Chapters';
 
 export function GettingStartedPanel() {
   const [step, setStep] = useState(0);
   const { closeWindow } = useWindowLayoutProvider();
+
+  const IntroductionSteps = useIntroductionSteps();
+  const NavigationSteps = useNavigationSteps();
+  const TimeSteps = useTimeSteps();
+  const ContentSteps = useContentSteps();
```

Comment:
Since these now are "regular variables" i think they should have camelcasing (`introductionSteps`)